blog/2024-04-30-mithril.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Mithril Team Update
+slug: 2024-04-30-mithril
+authors: jpraynaud
+tags: [mithril]
+hide_table_of_contents: false
+---
+
+## High level overview
+
+This week, the Mithril team prepared a new pre-release distribution [`2418.1-pre`](https://github.com/input-output-hk/mithril/releases/tag/2418.1-pre) which includes broader CPU support for pre-built binaries and a new memory allocator for the signer and aggregator nodes to prevent memory fragmentation. They also continued implementing the certification of Cardano transactions in Mithril networks and worked on scaling the signature and proof generation for `mainnet` by leveraging the compression of the transaction Merkle tree using sub-Merkle trees based on transaction block ranges during signature and proving. Additionally, they implemented a stream mechanism for importing transactions in the signer and aggregator stores.
+
+Finally, the team started implementing a global Mithril networks configuration file and kept investigating some unexpected error logs occurring on the Cardano node when the signer and aggregator connect to the mini-protocols.
```

Comment:
```suggestion
This week, the Mithril team prepared a new pre-release distribution [`2418.1-pre`](https://github.com/input-output-hk/mithril/releases/tag/2418.1-pre), which includes broader CPU support for pre-built binaries and a new memory allocator for the signer and aggregator nodes to prevent memory fragmentation. They also continued implementing the certification of Cardano transactions in Mithril networks and worked on scaling the signature and proof generation for `mainnet` by leveraging the compression of the transaction Merkle tree using sub-Merkle trees based on transaction block ranges during signature and proving. Additionally, they implemented a stream mechanism for importing transactions into the signer and aggregator stores.

Finally, the team started implementing a global Mithril networks configuration file and continued investigating some unexpected error logs occurring on the Cardano node when the signer and aggregator connect to the mini-protocols.
```